Makefile

```diff
@@ -8,15 +8,24 @@ PROW_JOBS_SCRAPER_TAG := $(or $(PROW_JOBS_SCRAPER_TAG),latest)
 
 install:
 	pip install .
+	$(MAKE) clean-install
```

Comment:
An improvement for the next PR is to do every job via tox, as it should operate in separate work dir

Reply:
yeah, we should review the build scripts